gatsby-browser.js

```diff
@@ -11,7 +11,7 @@ export const wrapRootElement = ({ element }) => (
 export const onClientEntry = () => {
   window[`ga-disable-${process.env.GOOGLE_ANALYTICS_ID}`] = true
 
-  if (Cookie.get('accept_cookies') === 'true' && Cookie.get('deny_analytics') === 'false') {
+  if (Cookie.get('accept_cookies') === 'true' && Cookie.get('accept_analytics') === 'false') {
```

Comment:
I guess it should be 
`Cookie.get('accept_analytics') === 'true'`

Reply:
Oh yes of course!